docs/pages/versions/unversioned/workflow/configuration.md

```diff
@@ -52,7 +52,9 @@ Valid values: `public`, `unlisted`
 
 ### `"sdkVersion"`
 
-**Required**. The Expo sdkVersion to run the project on. This should line up with the version specified in your package.json.
+__Deprecated__. The Expo sdkVersion to run the project on. This should line up with the version specified in your package.json.
+
+> This property is deprecated, it uses the `package.json`'s `expo` version to determine the SDK version. If you upgraded with `expo upgrade`, this property should have been removed.
```

Comment:
does it take the version from `package.json` or the version actually installed in `node_modules?

Reply:
It's currently only being read when the packages are installed. We should add support for reading from the `package.json` when packages aren't installed.